constraints/range_checker.air

```diff
@@ -40,14 +43,6 @@ ev range_checker_constraints([decoder[24], range_checker[2], chiplets[20]]) {
     let mv0 = chiplets[14]; # Memory value 0
     let mv1 = chiplets[15]; # Memory value 1
```

Comment:
Not from this PR, but should we name those variables `mem_d0` and `mem_d1`? With the current names, I had to go look in the docs there to remind myself which column needed a range check.

Reply:
yes, changed now